frontend/src/components/AppHeader/AppHeader.tsx

```diff
@@ -0,0 +1,20 @@
+import { AppHeaderWrapper } from './AppHeader.styled';
+import { PageHeader } from 'antd';
+
+export interface AppHeaderProps {
+  pageTitle?: string | null;
```

Comment:
jeśli jest question mark to chyba ten null jest niepotrzebny? @PiotrRynio?

Reply:
Poprawiłam